app/src/main/cpp/ControllerDelegate.h

```diff
@@ -32,6 +35,17 @@ enum class HandTrackingJoints {
   RingMetacarpal, RingProximal, RingIntermediate, RingDistal, RingTip,
   LittleMetacarpal, LittleProximal, LittleIntermediate, LittleDistal, LittleTip };
 
+struct Vector2f {
+  float x;
+  float y;
+};
```

Comment:
Seems like this is unused?

Reply:
Yes, this was used for the texture coords attributes, which were not used at the end. Will remove it.